File: espresso-macchiato/src/androidTest/java/de/nenick/espressomacchiato/test/views/BaseActivity.java
package de.nenick.espressomacchiato.test.views;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import de.nenick.espressomacchiato.test.R;

/** Simple activity with just a root layout */
public class BaseActivity extends AppCompatActivity {

    public static int rootLayout = R.id.layout_activity_base;

    /** Simple create method */
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_base);
    }
}

File: esp-internal-test-core/src/main/java/de/nenick/espressomacchiato/test/core/BaseActivity.kt
package de.nenick.espressomacchiato.test.core

import android.app.Activity
import android.os.Bundle

/** Simple activity with just a root layout  */
class BaseActivity : Activity() {

    companion object {
        var rootLayout: Int = R.id.layout_activity_base
    }

    /** Simple create method  */
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_base)
    }
}